components/SocialMediaIcons.jsx

```diff
@@ -3,9 +3,9 @@ import { FaFacebook, FaGithub, FaInstagram, FaLinkedin, FaYoutube } from "react-
 
 const ICON_SIZE = 30;
 
-const SocialMediaIcons = () => {
+const SocialMediaIcons = ({ mobile, desktop }) => {
   return (
-    <div className="flex items-start pt-5 text-[#FFD2DC] space-x-6">
+    <div className={`flex items-start pt-5 space-x-6 text-[${mobile}] md:text-[${desktop}]`}>
```

Comment:
Not sure if this is the best way to do it as it screws up the other component (colors missing):

<img width="488" alt="Screen Shot 2022-08-04 at 9 45 08 PM" src="https://user-images.githubusercontent.com/47438886/183003289-0e22e47e-6cfe-4da9-940b-d9a6a3f3e267.png">

I think what we want is something like this to override so we retain the classes original classes:

```js
const SocialMediaIcons = ({ classes }) => {
   ...
   <div className={`flex items-start pt-5 text-[#FFD2DC] space-x-6 ${classes}`}>
   ...
}
```

```js
<SocialMediaIcons classes="text-[#7055FD] md:text-[#FFFFFF]" />
```

Reply:
Wait that's weird because I can see the changes here. I'll take a look at it in a bit

![Screen Shot 2022-08-04 at 10 51 01 PM](https://user-images.githubusercontent.com/65151396/183003916-6d977597-4e1a-47a2-a762-0dc2bccfb7af.png)
![Screen Shot 2022-08-04 at 10 50 41 PM](https://user-images.githubusercontent.com/65151396/183003873-624bebae-0cd0-4e7b-bda6-46919da4d2dc.png)
![Screen Shot 2022-08-04 at 10 50 50 PM](https://user-images.githubusercontent.com/65151396/183003888-ae2cba78-3cda-4f87-bc99-5be0508d80c3.png)

Reply:
Okay I fixed it but it's not passing through the linter now

Reply:
Okay I'll take a look at it again tomorrow

Reply:
Should be good now